### PR TITLE
DTFS2-5448 - Accessibility Portal - Dashboard filters aria

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made: 

*  Added helper function to  convert filters to string
* Added to deals/facilities tab as aria-labels
* Ensured that only read out if active
* Updated tests

### CODE REVIEW

File: portal/templates/dashboard/_macros/dashboard-heading-and-tabs.njk [hunk 1]
```diff
@@ -7,6 +7,9 @@
   {% set successMessage = params.successMessage %}
   {% set user = params.user %}
   {% set tab = params.tab %}
+  {% set selectedFilters = params.selectedFilters %}
```
Comment (IgnatG) [inline portal/templates/dashboard/_macros/dashboard-heading-and-tabs.njk:10]: is `selectedFilters` being used?

File: e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js [hunk 2]
```diff
@@ -162,4 +162,14 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
 
     cy.url().should('eq', relative('/dashboard/deals/0'));
   });
+
+  it('renders the correct aria-labels based on filter selected', () => {
+    dashboardSubNavigation.deals().invoke('attr', 'aria-label').then((label) => {
+      expect(label).to.equal('deals: ,Filters selected: , Notice Type: , Automatic Inclusion Notice, Manual Inclusion Application, Status: , Draft, Ready for Checker\'s approval');
```
Comment (IgnatG) [inline e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js:168]: what does the `,` character do?

Comment (Zainzzkk) [reply e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js:168]: I assume it's leaving a slight pause which makes readout a bit clearer

Comment (IgnatG) [reply e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js:168]: ah. never mind. it's for pauses

File: portal/templates/dashboard/_macros/dashboard-heading-and-tabs.njk (hunk 1 above)
Comment (Zainzzkk) [reply portal/templates/dashboard/_macros/dashboard-heading-and-tabs.njk:10]: no, I can delete

**Review by IgnatG — APPROVED** [review]

**Pull request merged**